examples/integration/BUILD

```diff
@@ -15,16 +15,54 @@ load(
 
 exports_files(["README.md"])
 
-xcodeproj(
-    name = "xcodeproj",
-    associated_extra_files = ASSOCIATED_EXTRA_FILES,
-    build_mode = "xcode",
-    config = CONFIG,
-    extra_files = EXTRA_FILES,
-    project_name = "Integration",
-    scheme_autogeneration_mode = SCHEME_AUTOGENERATION_MODE,
-    schemes = get_xcode_schemes(),
-    tags = ["manual"],
-    top_level_targets = XCODEPROJ_TARGETS,
-    unfocused_targets = UNFOCUSED_TARGETS,
-)
+# Example of xcodeproj cache warming
+#
+# 1. Register a single macOS execution platform in `WORKSPACE`:
+#
+#      register_execution_platforms("@build_bazel_apple_support//platforms:macos_x86_64")
+#
+# 2. Create a `platform_mappings` file.
```

Comment:
I was unfamiliar with this [until Googling it](https://bazel.build/concepts/platforms#platform-mappings).

Is this something we could do for users via `--platform_mappings`? I admit as someone who <read above>, I don't know if I'll know if/when I need to update those mappings.

Reply:
Hmm. I expect users to be aware of this if they want to implement cache sharing between Intel and Apple silicon. I don’t think `rules_xcodeproj` should pass this on behalf of users because platform mappings is only a temporary API (until `rules_apple` fully supports building with platforms), and users may also use it for other kinds of builds. The `platform_mappings` file here is just an example, users may have their own definitions of `platform` targets and `platform_mappings` file, for instance, because of different `exec_properties` requirements from their RBE provider.